docs/Contribute/ContributeWrite.md

```diff
@@ -0,0 +1,142 @@
+# Writing for BTCPay Server
+
+- [Writing for BTCPay Server](#writing-for-btcpay-server)
+  - [Why writing matters](#why-writing-matters)
+  - [Requirements and recommended software](#requirements-and-recommended-software)
+    - [Requirements](#requirements)
+    - [Recommended software](#recommended-software)
+  - [Contribute to the Software Stack](#contribute-to-the-software-stack)
+  - [Contribute to Documentation](#contribute-to-documentation)
+  - [Contribute to the Directory](#contribute-to-the-directory)
+  - [Review other's work](#review-others-work)
+  - [Get help from the community](#get-help-from-the-community)
+
+
+## Why writing matters
+
+By giving essential and concise informations on software, or creating these documentation pages to help wrap your mind around the software stack, writers contributing to BTCPayServer are as important as any other contributer.
```

Comment:
BTCPayServer to BTCPay Server for consistency

Reply:
```suggestion
By giving essential and concise informations on software, or creating these documentation pages to help wrap your mind around the software stack, writers contributing to BTCPay Server are as important as any other contributor.
```